src/managers/EventManager.js

```diff
@@ -3,43 +3,27 @@
 import type { EventPublisher } from 'spark-protocol';
```

Comment:
Now EventManager becomes almost useless. what it does is just pipe the same functions to the publisher... But I think to keep it is good for make things consistent with other parts of our app( not use deviceServer inside controllers).
The only thing that I can add here: to expand subscribe method a bit with: if filterOptions.deviceID exist -> do checking for `doesUserHasAccess` and send HttpError if not.